vector/src/main/java/im/vector/adapters/VectorMessagesAdapter.java

```diff
@@ -1320,8 +1320,11 @@ private View getImageVideoView(int type, final int position, View convertView, V
 
 
             if (null != message) {
+                mHelper.hideStickerDescription(convertView);
```

Comment:
Could we hide it by default in the xml layout?

Reply:
this code is needed because of recycled views but you're right, it should be hidden by default.

Reply:
I agree to hide the description of the sticker by default in the xml. 
However, I have to keep the hideStickerDescription method when calling the image view, otherwise it remains displayed after the click. I therefore lose the behavior where the description appears when the message row is selected and disappears when it is deselected (via the booleans : isInSelectionMode & isSelected).